hypothesis-python/src/hypothesis/internal/compat.py

```diff
@@ -19,6 +19,7 @@
 import platform
 import sys
 import time
+import typing
 from typing import Tuple, Type  # noqa
```

Comment:
Is `noqa` here is still actual?

Reply:
Nope... in fact it looks like that whole statement can be deleted!